src/ch/blinkenlights/android/vanilla/LibraryPagerAdapter.java

```diff
@@ -852,6 +851,38 @@ public void onItemClick (AdapterView<?> parent, View view, int position, long id
 	}
 
 	/**
+	 * Perform usability-related actions on pager and contained lists, e.g. highlight current song
+	 * or scroll to it if opted-in
+	 * @param song song that is currently playing, can be null
+     */
```

Comment:
The code would only have to run ```view.setSelection()``` (or scroll...) in the ui thread, wouldn't it?

But well: We can keep it like this and change it later, if we notice that it causes UI frame drops. (But it probably won't as the code doesn't do anything heavy)